README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
```

Comment:
This feels vague. I might say,

`Druid's goal is to make it easy to write and deploy small and efficient desktop applications that work on most common platforms.'

Reply:
Sure, but then I'd like to see this item added to the goals list:
* Provide a smooth and polished user experience with pixel perfect design and no behavioral quirks.


smooth = no stutter
polished = care has been given to details
pixel perfect = widgets aren't being drawn willy-nilly in roughly the correct location, but instead in just the right pixels
behavioral quirks = application behaving differently under uncommon scenarios, like hot state not working when alt-tabbing out etc

Reply:
I tried to strike a middle ground on this one:

*Druid's goal is to make it easy to write and deploy high quality desktop
applications with a smooth and polished user experience on all common
platforms.*

as the summary, and

*Facilitate a smooth and polished user experience with no behavioral quirks*

as an additional point.

Reply:
I think if we already have the smooth & polish in the summary, then the extra point isn't needed.

Reply:
Well, you are right. I suppose it was a bit to late when I wrote that and my brain wasn't working properly anymore. I've removed the extra point.